person.rb

```diff
@@ -1,7 +1,8 @@
 require_relative './nameable'
+require_relative './rental'
 
 class Person < Nameable
-  attr_reader :id, :parent_permission
+  attr_reader :id, :parent_permission, :rentals
```

Comment:
- [ ] `rental` should be marked with `attr_accessor`, as we intend to both _access (get)_ as well as _modify (set)_ it. Kindly ensure to improve upon this aspect.